.env.testing.example

```
@@ -0,0 +1,7 @@
+# Settingsクラスのデフォルト値はFalseのため明示的に設定不要
+# DEBUG=false
+# testing 環境はシークレット不要（validate_production_secrets は production/staging のみ適用）
+API__BASE_URL=https://jsonplaceholder.typicode.com
+LOG__LEVEL=INFO
+# 外部API（JSONPlaceholder）にアクセスする統合テストの有効/無効
+TEST__EXTERNAL_API_ENABLED=true
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)